tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -31,34 +61,14 @@ class Jenkins {
 		
 		public bool Populating { get; private set; } = true;
 
+		public TestSelection TestSelection { get; set; } = DefaultTestSelection;
+		internal const TestSelection DefaultTestSelection =  TestSelection.MacOS | TestSelection.iOS | TestSelection.tvOS |
+		                                                   TestSelection.watchOS | TestSelection.iOSMSBuild |
+		                                                   TestSelection.Simulator | TestSelection.NonMonotouch |
+		                                                   TestSelection.Monotouch;
+
 		public IHarness Harness { get; }
-		public bool IncludeAll;
```

Comment:
You could have these properties become a getter that computes the value:

```csharp
public bool IncludeAll => (TestSelection & TestSelection.All) == TestSelection.All;
public bool IncludeBcl => (TestSelection & TestSelection.Bcl) == TestSelection.Bcl; 
...
```

that would make the diff significantly smaller, since you don't have to change the places that use these fields.